srcpkgs/libbitcoin-client/template

```diff
@@ -0,0 +1,56 @@
+# Template file for 'libbitcoin-client'
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+pkgname="libbitcoin-client"
+version=3.5.0
+revision=1
+short_desc="Bitcoin Client Query Library"
+makedepends+=" boost-devel"
+makedepends+=" icu-devel"
+makedepends+=" libbitcoin-protocol-devel"
+makedepends+=" libbitcoin-system-devel"
+makedepends+=" libpng-devel"
+makedepends+=" libsecp256k1-devel"
+makedepends+=" protobuf-devel"
+makedepends+=" qrencode-devel"
+makedepends+=" zeromq-devel"
+hostmakedepends+=" autoconf"
```

Comment:
same for hostmakedepends= but put it above makedepends=

our ordering is

hostmakedepends=
makedepends=
depends=
checkdepends=